lib/expr.ex

```diff
@@ -1498,8 +1498,14 @@ defmodule AshSql.Expr do
         List.wrap(bindings[:refs_at_path]) ++ relationship_path
       )
 
+    expression = if calculation.module.has_expression? do
```

Comment:
The real issue here is why we ever get to this code if the calculation does not have an expression. Something is going wrong if this is the case.